repos/spack_repo/builtin/packages/paraview/package.py

```diff
@@ -661,10 +677,17 @@ def use_x11():
         cmake_args.append("-DPARAVIEW_BUILD_SHARED_LIBS:BOOL=%s" % variant_bool("+shared"))
 
```

Comment:
See previous suggestion to just drop the variant.

```suggestion
use_viskores = spec.satisfies("+cuda") or spec.satisfies("+rocm") or spec.satisfies("+fides")
```